src/clsourcedependencies_parser.cc

```diff
@@ -0,0 +1,94 @@
+// /sourceDependencies files are only generated by MSVC. Don't bother including
+// the logic for JSON parsing in non-Windows builds.
+#ifdef _WIN32
+
+#include "clsourcedependencies_parser.h"
+
+#include "rapidjson/document.h"
+
+#include "clparser.h"
+#include "metrics.h"
+#include "util.h"
+
+using namespace rapidjson;
```

Comment:
Is this necessary? As someone who doesn't know RapidJSON at all, it would help to know which functions are from `rapidjson::`.

Reply:
Added.